lib/health_cards.rb

```diff
@@ -9,17 +9,18 @@
 require 'health_cards/private_key'
 require 'health_cards/public_key'
 require 'health_cards/health_card'
+require 'health_cards/payload'
 require 'health_cards/attribute_filters'
-require 'health_cards/card_types'
+require 'health_cards/payload_types'
 require 'health_cards/chunking_utils'
 require 'health_cards/errors'
 require 'health_cards/qr_codes'
 require 'health_cards/chunk'
 require 'health_cards/verifier'
 require 'health_cards/importer'
-require 'health_cards/covid_health_card'
-require 'health_cards/covid_immunization_card'
-require 'health_cards/covid_lab_result_card'
+require 'health_cards/covid_payload'
+require 'health_cards/covid_immunization_payload'
+require 'health_cards/covid_lab_result_payload'
```

Comment:
Should we have a `payload_types` subdirectory so that we can just import `health_cards/payload_types`? We have a decent amount of library files so it might be a good time to start grouping files together.

Reply:
Yes, I think this is a good idea. At some point we may want to consider splitting out the COVID specific stuff into its own gem so it can be updated independently of the underlying SHC implementation.